.github/workflows/publish.yml

```diff
@@ -22,35 +22,18 @@ jobs:
       pyi-ver: ${{ env.PYI_VER }}
       pyi-rel-ver: ${{ env.PYI_REL_VER }}
     steps:
-      - if: github.event_name == 'schedule' || github.event.inputs.workflow-ghidra-ver == 'latest'
-        name: Get Latest Ghidra Version
-        id: get_latest_ghidra_ver
-        uses: pozetroninc/github-action-get-latest-release@v0.6.0
-        with:
-          repository: NationalSecurityAgency/ghidra
-          excludes: prerelease, draft
-      - name: Set Ghidra Version from Latest
-        if: github.event_name == 'schedule' || github.event.inputs.workflow-ghidra-ver == 'latest'
-        id: format_ghidra_ver
-        run: |
-          echo "GHIDRA_VER=$(echo ${{steps.get_latest_ghidra_ver.outputs.release}} | cut -d_ -f2)" >> $GITHUB_ENV
+      - name: Set Ghidra Version to latest
+        run: echo "GHIDRA_VER=latest" >> $GITHUB_ENV
       - name: Set Ghidra Version from Input
         if: github.event_name == 'workflow_dispatch' && github.event.inputs.workflow-ghidra-ver != 'latest'
-        run: |             
-          echo "GHIDRA_VER=$(echo ${{github.event.inputs.workflow-ghidra-ver}})" >> $GITHUB_ENV
-      - name: Checkout repo
-        uses: actions/checkout@v3
+        run: echo "GHIDRA_VER=$(echo ${{github.event.inputs.workflow-ghidra-ver}})" >> $GITHUB_ENV
```

Comment:
Here I am unsure if `>> $GITHUB_ENV` is needed. I know that the GitLab CI has a construct where you can set the default value of an env variable, but override it with a user specified input to the workflow. That seems like the simpler way to do this